apps/hashdotai/docs/hash/working-with-data-in-pages/using-blocks-in-pages.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Using Blocks in Pages
+slug: hash/working-with-data-in-pages/using-blocks-in-pages
+objectId: ???
+description: Using Blocks in Pages
+---
+
+# Using Blocks in Pages {#using-blocks-in-pages}
+
+In order to view, create, or manipulate data or information in HASH, you’ll need to use Blocks.
+
+## How Blocks work with Entities, Links, and Entity Types {#how-blocks-work-with-entities-links-and-entity-types}
+
+Blocks can render and manipulate Entities, Links, Entity Types, and even Property, Data, and Link Types, though this is less common. Blocks can also create new Entities, Links, and Entity Types.
+
+You might use a Block which renders some Entities and Links which already exist in your HASH Workspace. Or you might use a Block to create new Entities of an Entity Type which doesn’t exist yet. Since Blocks are developed by third-parties, there is a huge range of possible data workflows which they enable.
+
+## The default Block: Paragraph Block {#the-default-block-paragraph-block}
+
+When you create a new Page in HASH, a Paragraph Block will automatically be inserted at the top of the page and your cursor will be placed inside the block, ready to type. HASH does this so that Pages can be used to quickly capture some writing.
+
+The Paragraph Block creates a Text Entity in your HASH Workspace. The Text Entity conforms to the Text Entity Type, which is a non-editable system type. As you write in the Paragraph Block, the ‘Tokens’ Property on the Text Entity is updated to capture your text.
```

Comment:
```suggestion
The paragraph block creates a `Text` entity in your workspace. The `Text` entity conforms to the `Text` entity type, which is a non-editable _system type_. As you write in the paragraph block, the ‘Tokens’ property on the `Text` entity is updated to capture your text.
```

Do we need to talk about system types? It seems like extra/unnecessary information for most users. As, frankly, does explaining how the paragraph block works at all (e.g. 'tokens' property).

Reply:
@jtewright @CiaranMn @Alfred-Mountfield for thoughts.

Reply:
While reading it, I did feel like it was slightly strange to describe the inner workings of the blocks and types. But if this is supposed to be a guiding walk-through of blocks in HASH, how they work with entity types, etc. then our text blocks are going to be fairly confusing without us explaining them. (Ideally we'd just avoid them and walk through something much simpler).

The hook service, the `Tokens`, ProseMirror, all of that makes them pretty confusing and if we're trying to say "hey you can insert a block", and then "you can go look at / edit types associated with your block", then as a user I'd try and look at the paragraph block and.. :panic:

Reply:
I added this bit about the Paragraph block and Text type cos it was the only block/type which was working and I ended up in the Type Editor sidebar for the Text type, so thought I should explain it.

Reply:
I feel like we should just pick another block that will work once updated to 0.3, and use this. Framing it in terms of `Text` is just a bit confusing/unnecessarily abstract for most people thinking about what "Entities" and "Types" are (IMO).

Reply:
@jtewright Could you pick another block for this? e.g. `Countdown`